<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeFullWidget</class>
 <widget class="QWidget" name="RecipeFullWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecipeName</string>
  </property>
  <property name="styleSheet">
   <string notr="true">       background-color:rgb(222, 255, 238);</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">       background-color:#e7f0f5;
       border-radius:0px;
       border:1px solid #2d4ba3;
       color:#000000;
       font-family:Arial;
       font-size:19px;
       padding:0px 0px;
       text-decoration:none;
</string>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_dscr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">       background-color:#e7f0f5;
       border-radius:0px;
       border:1px solid #2d4ba3;
       color:#000000;
       font-family:Arial;
       font-size:19px;
       padding:0px 0px;
       text-decoration:none;
</string>
   </property>
   <property name="text">
    <string>Описание</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>371</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">       background-color:#e7f0f5;
       border-radius:0px;
       border:1px solid #2d4ba3;
       color:#000000;
       font-family:Arial;
       font-size:19px;
       padding:0px 0px;
       text-decoration:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_text">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>531</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">       background-color:#e7f0f5;
       border-radius:0px;
       border:1px solid #2d4ba3;
       color:#000000;
       font-family:Arial;
       font-size:19px;
       padding:0px 0px;
       text-decoration:none;
</string>
   </property>
   <property name="text">
    <string>Текст рецепта</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_chg">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить рецепт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">       background-color:#e7f0f5;
       border-radius:0px;
       border:1px solid #2d4ba3;
       color:#000000;
       font-family:Arial;
       font-size:19px;
       padding:0px 0px;
       text-decoration:none;
</string>
   </property>
   <property name="text">
    <string>Время</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_ingr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">       background-color:#e7f0f5;
       border-radius:0px;
       border:1px solid #2d4ba3;
       color:#000000;
       font-family:Arial;
       font-size:19px;
       padding:0px 0px;
       text-decoration:none;
</string>
   </property>
   <property name="text">
    <string>Ингредиенты</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_name_ingr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">       background-color:#e7f0f5;
       border-radius:0px;
       border:1px solid #2d4ba3;
       color:#000000;
       font-family:Arial;
       font-size:19px;
       padding:0px 0px;
       text-decoration:none;
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
